### PR TITLE
the field data refinement typed tests now work for 1, 2 & 3d

### PR DESCRIPTION
**Pull request by rochSmets**
#363

### CODE REVIEW

File: pyphare/.idea/inspectionProfiles/profiles_settings.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (nicolasaunai) [inline pyphare/.idea/inspectionProfiles/profiles_settings.xml:1]: add .idea to the gitignore

**Review by nicolasaunai — DISMISSED** [review]
LGTM, can you add a comment, for instance at the location where input.txt files are read, or even better, in the input.txt files themselves if the samrai syntax allows for comments, that the test assumes the refinement boxes are such that the refined patches should NOT have any ghost beyond the coarest level domain boundaries, because periodicity would apply and ghost nodes would NOT have the value expected from the linear function.

**Review by nicolasaunai — COMMENTED** [review]
LGTM, can you add a comment, for instance at the location where input.txt files are read, or even better, in the input.txt files themselves if the samrai syntax allows for comments, that the test assumes the refinement boxes are such that the refined patches should NOT have any ghost beyond the coarest level domain boundaries, because periodicity would apply and ghost nodes would NOT have the value expected from the linear function.

**Comment by nicolasaunai** [comment]
actually @rochSmets #363 mentions that we could have a different affine function for each quantity that is refined, what do you think? doing that would allow us to be sure patchdatas are not mixed with each others while refining.

File: pyphare/.idea/inspectionProfiles/profiles_settings.xml (hunk 1 above)
Comment (rochSmets) [reply pyphare/.idea/inspectionProfiles/profiles_settings.xml:1]: such comments have been added

**Comment by rochSmets** [comment]
I added the the dataId value in the coeff for affineFill, so all components have different coefficients.
I guess I answered to all your comments guys,

**Review by nicolasaunai — COMMENTED** [review]
remove .idea/* files and we merge

**Review by nicolasaunai — APPROVED** [review]

**Review by PhilipDeegan — APPROVED** [review]

**Pull request merged**